src/app/econ/supply-demand/lesson/SupplyDemandLesson.tsx

```diff
@@ -0,0 +1,807 @@
+            />
+            <p style={{ marginTop: "1rem", fontSize: "14px" }}>
+              This lesson is based off of videos by{" "}
+              <a
+                style={{ color: "blue", textDecoration: "underline" }}
+                href="https://www.youtube.com/@JacobAClifford"
+              >
+                Jacob Clifford
+              </a>
+            </p>
+          </center>
+        </div>
+      </div>
+    </Block>,
+    <Block color="blue" title="Graphs">
+      <img
+        src="https://upload.wikimedia.org/wikipedia/commons/f/f2/Simple_supply_and_demand.svg"
+        alt="supply and demand graph"
+      />
+
+      <ColorBox color="blue">
+        <p>
+          <Emphasize>This is the supply and demand graph,</Emphasize>
+          <br />
+          <br />
+          which shows the relationship between the price of a good and the
+          quantity. The x-axis is the quantity of the good, and the y-axis is
+          the price of the good.
+          <br />
+          <br />
+          The <Emphasize>Demand curve</Emphasize> is the line where it is
+          sloping downwards and the <Emphasize>Supply curve</Emphasize> is the
+          line where it is sloping upwards.
+          <br />
+          <br />
+          Now let's dig deeper to see why this is the case:
+        </p>
+      </ColorBox>
+    </Block>,
+
+    <Block color="blue" title="Demand">
+      <p>
+        Demand means how much people want to buy something.
+        <br />
+        <Emphasize>👉 Example: </Emphasize> If ice cream is cheap on a hot day,
+        lots of people want it!{" "}
+      </p>
+      <ColorBox color="blue">
+        <p>
+          The law of demand states that{" "}
+          <Emphasize>
+            as price goes up, the amount that people want to buy
+          </Emphasize>{" "}
+          (Quantity Demanded) <Emphasize> goes down</Emphasize>, and vice versa.
+        </p>
+        <div>
+          <Emphasize>
+            <u>Reasons: </u>
+          </Emphasize>
+          <ul>
+            <li>
+              <Collapsible
+                ExampleContent={
+                  <>
+                    <Emphasize>
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Example:{" "}
+                    </Emphasize>{" "}
+                    If ice cream is too expensive, they will buy a candy bar
+                    instead
+                  </>
+                }
+              >
+                <Emphasize>- Substitution effect: </Emphasize> Consumers switch
+                to alternatives.
+              </Collapsible>
+            </li>
+            <li>
+              <Collapsible
+                ExampleContent={
+                  <>
+                    <Emphasize>
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Example:{" "}
+                    </Emphasize>{" "}
+                    If they have more money to spend, they will get fine dining
+                    instead of McDonald's.
```

Comment:
super nit: This is a finer point that we discuss in teaching pedagogy workshops, but I'd avoid using specifics when making allusions to wealth. For example, this example creates the ideological mapping { Have more money -> Eat fine dining, not McDonalds }, but what it actually does is it also leaves room for the reverse mapping { My family eats at McDonalds <-> We don't have much money }

Since education can be such an example-driven ordeal, it can be really hard to come up with examples that avoid these kinds of connections